src/tests/topp/THIRDPARTY/CometAdapter_2_out.idXML

```diff
@@ -1,105 +1,105 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <?xml-stylesheet type="text/xsl" href="https://www.openms.de/xml-stylesheet/IdXML.xsl" ?>
 <IdXML version="1.5" xsi:noNamespaceSchemaLocation="https://www.openms.de/xml-schema/IdXML_1_5.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
-	<SearchParameters id="SP_0" db="/home/hr/openmsall/source/OpenMS/src/tests/topp/THIRDPARTY/CometAdapter_2_in.fasta" db_version="" taxonomy="" mass_type="monoisotopic" charges="" enzyme="trypsin" missed_cleavages="1" precursor_peak_tolerance="3" precursor_peak_tolerance_ppm="false" peak_mass_tolerance="0.50025" peak_mass_tolerance_ppm="false" >
+	<SearchParameters id="SP_0" db="/workspace/OpenMS/src/tests/topp/THIRDPARTY/CometAdapter_2_in.fasta" db_version="" taxonomy="" mass_type="monoisotopic" charges="" enzyme="trypsin" missed_cleavages="1" precursor_peak_tolerance="3" precursor_peak_tolerance_ppm="false" peak_mass_tolerance="0.50025" peak_mass_tolerance_ppm="false" >
 	</SearchParameters>
-	<IdentificationRun date="2017-12-07T16:39:44" search_engine="Comet" search_engine_version="" search_parameters_ref="SP_0" >
+	<IdentificationRun date="2020-03-27T19:10:22" search_engine="Comet" search_engine_version="" search_parameters_ref="SP_0" >
 		<ProteinIdentification score_type="" higher_score_better="true" significance_threshold="0" >
-			<ProteinHit id="PH_0" accession="prot1" score="0" sequence="" >
+			<ProteinHit id="PH_0" accession="prot1" score="0.0" sequence="" >
 			</ProteinHit>
-			<ProteinHit id="PH_1" accession="prot3" score="0" sequence="" >
+			<ProteinHit id="PH_1" accession="prot3" score="0.0" sequence="" >
 			</ProteinHit>
-			<ProteinHit id="PH_2" accession="prot2" score="0" sequence="" >
+			<ProteinHit id="PH_2" accession="prot2" score="0.0" sequence="" >
 			</ProteinHit>
-			<ProteinHit id="PH_3" accession="prot4" score="0" sequence="" >
+			<ProteinHit id="PH_3" accession="prot4" score="0.0" sequence="" >
 			</ProteinHit>
-			<ProteinHit id="PH_4" accession="prot5" score="0" sequence="" >
+			<ProteinHit id="PH_4" accession="prot5" score="0.0" sequence="" >
 			</ProteinHit>
-			<ProteinHit id="PH_5" accession="prot6" score="0" sequence="" >
+			<ProteinHit id="PH_5" accession="prot6" score="0.0" sequence="" >
 			</ProteinHit>
-			<ProteinHit id="PH_6" accession="prot10" score="0" sequence="" >
+			<ProteinHit id="PH_6" accession="prot10" score="0.0" sequence="" >
 			</ProteinHit>
-			<UserParam type="stringList" name="spectra_data" value="[/Users/builder/jenkins/ws/openms_pr_ci/c797036f/source/src/tests/topp/THIRDPARTY/spectra_comet.mzML]"/>
+			<UserParam type="stringList" name="spectra_data" value="[CometAdapter_2_prepared.mzML]"/>
```

Comment:
Is this correct? Why did that change?

Reply:
hmm from this line it seems correct. Though I think it should annotate the absolute path.
 https://github.com/OpenMS/OpenMS/blob/9d2282c40359c6cbd8e13e8bd220c8737b4e377a/src/tests/topp/THIRDPARTY/third_party_tests.cmake#L162

Reply:
TODO:
in1:  C:\jenkins\ws\openms\PR\bldtst\031d6063\build\src\tests\topp\CometAdapter_1_out1.tmp   (line: 8, position/column: 4/26)
			<!
			 UserParam type="stringList" name="spectra_data" value="[C:/jenkins/ws/openms/PR/bldtst/031d6063/source/src/tests/topp/THIRDPARTY/spectra_comet.mzML]"/>

in2:  C:\jenkins\ws\openms\PR\bldtst\031d6063\source\src\tests\topp\THIRDPARTY\CometAdapter_1_out.idXML   (line: 8, position/column: 4/26)
			<!
			 ProteinHit id="PH_0" accession="test2_rev" score="0.0" sequence="" >